### PR TITLE
Select passcode text when invalid

### LINKED ISSUE #119: Highlight user entered value if input is invalid
#### Describe the bug
1. Go to URL /forms-and-validation/fixed-length-passcode
2. Enter invalid passcode `111111`.
3. It gives error `invalid passcode`.

#### What is the expected behavior?
 1. The field should active again
 2. Select / highlight the entire user-entered passcode.
image

#### What are the steps to reproduce?
1. Go to /forms-and-validation/fixed-length-passcode
2. Enter invalid passcode

Note: Same behavior should apply for `Verify on submit` code example URL - `/forms-and-validation/verify-on-submit`

### LINKED ISSUE #129: Update password validation example to remove title on small view
#### Describe the bug
Currently the page title is displayed when viewing example on small view
#### What is the expected behavior?
Title should be hidden
#### What are the steps to reproduce?
1. Go to...
2. Click on...

#### Screenshots or links to minimum reproduction example
image

### PR DESCRIPTION
**Pull request by huayunh**
Fixes #119, fixes #129

#### Screenshots:

https://user-images.githubusercontent.com/8997218/134367282-f08fb2d8-7f10-45a1-8fbc-a402e50c19c4.mov

Just waiting for the latest theme update to make the selection color red.

### CODE REVIEW

File: src/app/pages/forms-and-validation/password/password.component.html [hunk 1]
```diff
@@ -7,14 +7,6 @@
 
 <section class="change-password-section">
     <mat-card class="change-password-card">
-        <mat-card-header>
```
Comment (emclaug2) [inline src/app/pages/forms-and-validation/password/password.component.html:10]: image

Need to keep the title on non-mobile.  Maybe just put a `*ngIf="!isSmall"` on it.

File: src/app/pages/forms-and-validation/fix-length-passcode/fix-length-passcode.component.ts [hunk 2]
```diff
@@ -91,6 +91,7 @@ export class FixLengthPasscodeComponent implements OnInit, AfterViewInit {
                     this.showLoading = false;
                     this.showDoneIcon = false;
                     this.passcodeInput.nativeElement.focus();
+                    this.passcodeInput.nativeElement.select();
```
Comment (ektasawant) [inline src/app/pages/forms-and-validation/fix-length-passcode/fix-length-passcode.component.ts:94]: Could you please add this feature in verify on submit? https://github.com/pxblue/angular-design-patterns/blob/dev/src/app/pages/forms-and-validation/verify-on-submit/verify-on-submit.component.ts#L73.
As per design need to highlight invalid input in `verify on submit` form.

**Comment by huayunh** [comment]
All fixed

https://user-images.githubusercontent.com/8997218/134384745-6abcb916-7919-47b5-ac4d-e4fb3a4e6c9e.mov

**Comment by emclaug2** [comment]
Ahh, you'll need to hide the title when the viewport <= Handset.

use-handset

**Comment by huayunh** [comment]
Fixed & tweaked the mobile style a little bit
https://user-images.githubusercontent.com/8997218/134409744-5e47e28f-e1d7-458c-82f1-9a0ae39194ad.mov

**Review by emclaug2 — APPROVED** [review]

**Pull request merged**